src/main/resources/driven-adapter/async-event-bus/reactive-events-gateway.java.mustache

```diff
@@ -1,45 +1,42 @@
-package {{package}}.event;
+package {{package}}.events;
 
-import {{package}}.model.event.gateways.EventsGateway;
+import {{package}}.model.events.gateways.EventsGateway;
 {{#lombok}}
 import lombok.RequiredArgsConstructor;
 import lombok.extern.java.Log;
 {{/lombok}}
 import org.reactivecommons.api.domain.DomainEvent;
 import org.reactivecommons.api.domain.DomainEventBus;
 import org.reactivecommons.async.impl.config.annotations.EnableDomainEventBus;
-import org.springframework.stereotype.Component;
 import reactor.core.publisher.Mono;
 
 import java.util.UUID;
+{{#lombok}}
 import java.util.logging.Level;
+{{/lombok}}
 
 import static reactor.core.publisher.Mono.from;
 
-@Component
-@EnableDomainEventBus
 {{#lombok}}
 @Log
 @RequiredArgsConstructor
 {{/lombok}}
```

Comment:
maybe if you change the order yo need only one lombok validation

Reply:
could be, but the imports are sorted and arranged by package